components/atoms/PhaseBanner.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays the PhaseBanner on the page
+ */
+
+export const PhaseBanner = ({ phase, children }) => {
+  return (
+    <div className="bg-circle-color">
+      <div className="block sm:flex py-4 layout-container">
+        <span className="font-body text-xs text-circle-color font-extrabold flex-shrink-0 bg-white px-4 py-1 my-auto leading-6">
+          {phase}
+        </span>
+        <p className="font-body text-xs mt-5 sm:mt-auto text-white sm:ml-4 pt-1 my-auto">
```

Comment:
Looking real sharp. All I would recommend is that we vertically center the text within the space that it has so it lines up right along the horizontal center axis of the "Test site" block.
![image](https://user-images.githubusercontent.com/31868510/125850954-10a3bb75-1625-483c-b7c9-a2c92c4e28e1.png)

Reply:
`align-middle` did not work so I added bottom padding of 1px at large screens. it does the trick. :)